package.json

```diff
@@ -131,6 +131,11 @@
 					"type": "boolean",
 					"default": false,
 					"description": "Automatically start ReScript's code analysis."
+				},
+				"rescript.settings.binaryPath": {
+					"type": "string",
+					"default": "",
+					"description": "Path to the ReScript binary. You can use it if you haven't or don't want to use the installed ReScript binary from node_modules in your project."
```

Comment:
Can you update the description: it's the path to the directory where binaries are.

Reply:
done